.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]

```

Comment:
Create Makefile for run command like _lint_, _format_
https://antonz.ru/makefile/

Find a windows analogue